spec/features/idv/doc_auth/verify_step_spec.rb

```diff
@@ -203,20 +203,22 @@
     it 'does not perform the state ID check' do
       allow(IdentityConfig.store).to receive(:aamva_sp_banlist_issuers).
         and_return('["urn:gov:gsa:openidconnect:sp:server"]')
+      expect_any_instance_of(Idv::Agent).
```

Comment:
`expect_any_instance_of` is not my favorite, but given that these are feature specs we don't have much of a choice 😬  and I'd rather have this than stub `.new` 🤷

Reply:
> `expect_any_instance_of` is not my favorite

Curious: Could you elaborate on why you'd avoid it?

Reply:
Ah, I see what you mean now, and I see how feature specs in particular may not be particularly well suited for the ideal implementation. Thanks for explaining 👍